src/tests/nf-token-metadata-integrity.js

```diff
@@ -0,0 +1,47 @@
+const { expect } = require('chai');
+
+describe('nf-token-metadata-integrity', function() {
+  let nfToken, owner, bob;
+  const schemaDigest = '0x9c22ff5f21f0b81b113e63f7db6da94fedef11b2119b4088b89664fb9a3cb658';
```

Comment:
I'd be happy to see `digest` values to be created from actual source included in the test case.

Reply:
Done. Check if ok.